website/src/pages/index.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Home
+description: A fully-featured GraphQL gateway framework
```

Comment:
added description from guild blog, instead of 
<img width="479" alt="image" src="https://github.com/user-attachments/assets/1bb302b1-6380-461b-a106-138988a02317">